asreview/webapp/api.py

```diff
@@ -714,49 +703,112 @@ def api_set_algorithms(project_id):  # noqa: F401
 
 @bp.route('/project/<project_id>/start', methods=["POST"])
 def api_start(project_id):  # noqa: F401
-    """Start training the model
+    """Start training of first model or simulation.
     """
 
-    # start training the model
-    py_exe = _get_executable()
-    run_command = [
-        py_exe, "-m", "asreview", "web_run_model", project_id,
-        "--label_method", "prior"
-    ]
-    subprocess.Popen(run_command)
+    project_config = get_project_config(project_id)
+
+    # the project is a simulation project
+    if project_config["mode"] == PROJECT_MODE_SIMULATE:
+
+        logging.info("Starting simulation")
+
+        try:
+            simulation_id = uuid.uuid4().hex
+            datafile = get_data_file_path(project_id)
+            state_file = get_simulation_ready_path(
+                project_id, simulation_id
+            )
+
+            logging.info("Project data file found: {}".format(datafile))
+
+            add_simulation_to_project(project_id, simulation_id)
+
+            # start simulation
+            py_exe = _get_executable()
+            run_command = [
+                py_exe, "-m", "asreview", "simulate",
+                datafile, "--state_file", state_file
+            ]
+            subprocess.Popen(run_command)
+
+        except Exception as err:
+            logging.error(err)
+            message = f"Failed to get data file. {err}"
+            return jsonify(message=message), 400
+
+    # the project is an oracle or explore project
+    else:
+
+        logging.info("Train first iteration of model")
+        try:
+            # start training the model
+            py_exe = _get_executable()
+            run_command = [
+                py_exe, "-m", "asreview", "web_run_model", project_id,
+                "--label_method", "prior"
+            ]
+            subprocess.Popen(run_command)
+
+        except Exception as err:
+            logging.error(err)
+            return jsonify(message="Failed to train the model."), 500
 
     response = jsonify({'success': True})
     response.headers.add('Access-Control-Allow-Origin', '*')
     return response
 
 
-@bp.route('/project/<project_id>/model/init_ready', methods=["GET"])
+@bp.route('/project/<project_id>/ready', methods=["GET"])
```

Comment:
The corresponding test for this API in `test_api.py` should also be changed.

Reply:
Thanks